packages/flutter/test/cupertino/context_menu_action_test.dart

```diff
@@ -61,4 +87,22 @@ void main() {
     await tester.pump();
     expect(_getDecoration(tester).color, _kBackgroundColor);
   });
+
+  testWidgets('icon and textStyle colors are right out of the box',(WidgetTester tester) async {
```

Comment:
Nitpick: Put a space after the comma in these 3 tests, so it would be: `...box', (WidgetTester...`

Reply:
Also, I think it would be more clear if you said "correct" instead of "right" in all 3 of the descriptions.

Reply:
Okay, I will make the changes asap,  in about two hours. Thanks for the heads-up, I agree.

Reply:
Thanks!  No big deal, but I appreciate the small changes.  It looks like the tree is closed right now and we can't merge, so no rush.

Reply:
Alright, pushed the changes already nevertheless.